lib/src/components/NxTab/NxTab.tsx

```diff
@@ -39,7 +39,7 @@ const NxTab = function NxTabElement(props: Props) {
     }
   }
 
-  return <li role="tab" id={id} aria-selected={selected} className={classNames} onKeyPress={handleKeyPress} onClick={handleClick} {...attrs}/>;
+  return <li role="tab" id={id} aria-selected={selected} className={classNames} onKeyPress={handleKeyPress} onClick={handleClick} tabIndex={tabIndex} {...attrs}/>;
```

Comment:
A cleaner way would be to remove the `tabIndex` local var and just rely on the `...attrs` to override the default:
```suggestion
  return <li role="tab" id={id} aria-selected={selected} className={classNames} onKeyPress={handleKeyPress} onClick={handleClick} tabIndex={0} {...attrs}/>;
```

Reply:
Cleaner maybe, but I think it hides that the `tabIndex` of 0 would be replaced by the `...attrs`. Looking at the code I wouldn't have thought that I could override the `tabIndex` even though the de-structuring would do that.